src/transformers/training_args.py

```diff
@@ -1589,6 +1573,9 @@ def __post_init__(self):
             )
             self.use_cpu = self.no_cuda
 
+        if self.mixed_precision_config is None:
+            self.mixed_precision_config = {}
```

Comment:
Make sure to replace the existing logic (e.g bf16_full_eval) with `mixed_precision_config`